src/firefox/manifest.json

```diff
@@ -18,7 +19,7 @@
   },
   "browser_action": {
     "default_icon": {
-      "32": "images/firefox-launch/32grey.png"
```

Comment:
I am wondering if we should update the add-on ID in `bss.gecko.id`, WDYT? It is currently "firefox-launch@mozilla.com" but we started. to rename everything to "firefox-bridge".

Reply:
No objections, was just a placeholder I overlooked.